src/Components/Sizing/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
```

Comment:
When making a new component I've generally a `ComponentName.js` file and then import and export it in an `index.js` file.  Mainly for the fact that it helps locate the right file when you've got multiple files open in your IDE.

Reply:
I've done that now